### PR TITLE
docs for redis authentication 

### PR DESCRIPTION
**Pull request by olevchyk**
see https://github.com/zalando-zmon/zmon-worker/pull/389

### CODE REVIEW

File: docs/user/check-ref/redis_wrapper.rst [hunk 1]
```diff
@@ -10,6 +10,16 @@ Read-only access to Redis servers is provided by the :py:func:`redis` function.
...
 
+    :param host: Redis host.
+    :type host: str
+
+    :param password:  If set - enables authentication to the destination server with the password provided.
+    :type password: str
+
+.. note::
+
+    You can also use ``plugin.redis.password`` plugin confirguration for zmon appliance to enable password auth for this plugin.
```
Comment (mohabusama) [inline docs/user/check-ref/redis_wrapper.rst]: ZMON appliance is a special form of deployment. This part is mainly concerned with Worker plugins. So I would rather mention that redis password can be configured.

File: docs/user/check-ref/redis_wrapper.rst [hunk 2]
```diff
@@ -10,6 +10,16 @@ Read-only access to Redis servers is provided by the :py:func:`redis` function.
     of the current entity's ``host`` attribute, and :samp:`{<port>}` is the given port (default ``6379``). See
     below for a list of methods provided by the returned connection object.
 
+    :param host: Redis host.
+    :type host: str
+
+    :param password:  If set - enables authentication to the destination server with the password provided.
```
Comment (mohabusama) [inline docs/user/check-ref/redis_wrapper.rst]: Please add `Default is None`, to avoid confusion for non-auth redis servers.

Comment (olevchyk) [reply docs/user/check-ref/redis_wrapper.rst]: added

File: docs/user/check-ref/redis_wrapper.rst (hunk 1 above)
Comment (olevchyk) [reply docs/user/check-ref/redis_wrapper.rst]: tnx, done, please take a look.

**Comment by mohabusama** [comment]
Thanks!
Will merge with a new zmon-worker release.

**Comment by vetinari** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Review by mohabusama — APPROVED** [review]

**Pull request merged**